lib/OpenQA/Utils.pm

```diff
@@ -166,6 +168,23 @@ sub testcasedir {
     return $dir;
 }
 
+sub gitrepodir {
```

Comment:
Please add documentation to this function. I would like to know what it is intended to do. Please use POD format

Reply:
*this change is ready for review*